css-three-column-layout/styles.css

```diff
@@ -0,0 +1,74 @@
+.background-color {
+  background-color: #0097e0;
+}
+
+.container {
+  margin: auto;
+  max-width: 1000px;
+}
+
+.row {
+  display: flex;
+}
+
+.column-full {
+  flex-basis: 100%;
+}
+
+.column-third {
+  flex-basis: calc(100% / 3);
+}
+
+.text-color {
+  color: white;
+}
+
+.text-style {
+  font-family: Lato, sans-serif;
+}
+
+.center-text {
+  text-align: center;
+}
+
+.justify-center {
+  justify-content: center;
+}
+
+.header {
+  margin-bottom: 0;
+}
+
+.info {
+  line-height: 18px;
+  font-size: 12px;
+  margin-top: 10px;
+  color: lightgray;
+}
+
+.first-section {
+  padding-left: 150px;
+}
+
+.second-section {
+  padding: 0 45px;
+}
+
+.third-section {
+  padding-right: 155px;
+}
```

Comment:
Instead of ding this, I would apply a class that adds padding to the left and right on the `row` holding these `column` elements. You can add a touch of padding on the `columns` themselves too, to put space between them.